frontend/src/util/index.tsx

```diff
@@ -334,3 +336,23 @@ export const floatingMenuProps = (isDark: boolean) => ({
     borderWidth: isDark ? 1 : 0,
     backgroundColor: isDark ? COLORS.neutral15 : COLORS.neutral05,
 }) as const;
+
+
```

Comment:
(snippet unrelated)

Currently we have a weird ` h2: { display: "none" }` where we use `TrackInfo` in the downloads menu. It would be cleaner to pull out the main content of `TrackInfo` into a component in `ui/Video` or sth like that. The `<section>` and `<h2>` the stays in `TechnicalDetails` and isn't in the downloads menu then.